src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+// categories.js
+
+// Actions
+const CHECKING_STATUS = 'CHECKING_STATUS';
```

Comment:
- [ ] I think here it would be great if you create the action in the duck's patterns as existed in the requirements Write your book's actions and reducer (in /src/redux/books/books.js) using the [ducks pattern](https://github.com/erikras/ducks-modular-redux).

- You can write something like this 
> const CHECK_STATUS = bookstore/categories/CHECK_STATUS